ampersand-state.js

```diff
@@ -1,13 +1,35 @@
 /*$AMPERSAND_VERSION*/
-var _ = require('underscore');
-var BBEvents = require('backbone-events-standalone');
+var uniqueId = require('amp-unique-id');
+var extend = require('amp-extend');
+var escape = require('amp-escape');
+var each = require('amp-each');
+var clone = require('amp-clone');
+var bind = require('amp-bind');
+var unique = require('amp-unique');
+var flatten = require('amp-result');
```

Comment:
Happy to make a change, but I'm afraid I don't know what you mean by "doesn't look right"?